README.arm64_build_on_amd64.md

```diff
@@ -0,0 +1,322 @@
+# **ARM64 Build on AMD64 for SONiC BuildImage**
+
+## **Table of Contents**
+
+1. [About This Guide](#about-this-guide)  
+2. [Hardware Specifications](#hardware-specifications)  
+3. [Test Server Details](#test-server-details)  
+4. [Pre-requisites](#pre-requisites)  
+5. [Build Procedure](#build-procedure)  
+6. [Expected Errors and Workarounds](#expected-errors-and-workarounds)  
+7. [Console Output](#console-output)  
+8. [Reference Links](#reference-links)  
+
+## **About This Guide**
+
+This document provides a detailed guide for building the SONiC BuildImage for the ARM64 architecture on an AMD64 system. During this process, build errors were encountered. The steps below outline the necessary hardware specifications, pre-requisites, build procedures, and solutions to known issues to ensure a successful build. This guide is based on the build conducted using both Debian 11 (Bullseye) Virtual Machine and Ubuntu 22.04.2 Host server.
```

Comment:
Solutions and workarounds for the common build errors that may be encountered are provided below.